doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
+To upgrade an existing instance of `net.Socket` to a 
+`tls.TLSSocket`, pass it to `tls.connect()` as 
+the socket option: 
+
+```js
+const { Socket } = require('net');
+const tls = require('tls');
+const sock = new Socket();
+const secureSock = tls.connect({ port: 6697, host: 'irc.freenode.net' }, () => {
```

Comment:
In the documentation? I wouldn’t worry about that, although example.org:443 might be a somewhat more natural choice?